photo_points/Views/Admin/PhotoStream.cshtml

```diff
@@ -5,42 +5,42 @@
     Layout = "../Shared/_Layout.cshtml";
 }
 
-@*<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
-<link rel="stylesheet" href="../wwwroot/css/photo-card.css" />
-<link rel="stylesheet" href="../wwwroot/css/body.css" />*@
 
 <h2>PhotoStream</h2>
 
+
 @*@foreach (photo_points.Models.PhotoPoint photoPoint in @Model) {*@
 
-@for (var i = 0; i < 5; i++) {
 
-<div class="container card-body-sm">
-    <div class="row">
-        <img class="card-img-sm" src="~/images/fern.jpg" />
-    </div>
-    <div class="row card-sm-content">
-        <h3>Photopoint Name</h3>
-        <div class="col-sm-10">
-            <div class="row">
-                <h6>Jan 15, 2013</h6>
-            </div>
+@for (var i = 0; i < 5; i++)
+{
 

```

Comment:
This was me trying to implement a modal, it didn't work, I can go back and remove some of that messy code.